### PR TITLE
Charge: Add `customer` and `created` to the list API

### PR DESCRIPTION
**Pull request by H--o-l**
Made from Stripe documentation:
https://stripe.com/docs/api/charges/list#list_charges-customer

--- 

~~This PR is based on top of the other PR https://github.com/adrienverge/localstripe/pull/141~~

### CODE REVIEW

File: localstripe/resources.py [hunk 1]
```diff
@@ -475,6 +475,33 @@ def amount_refunded(self):
...
+            if type(created) is dict:
+                assert len(created.keys()) == 1 and \
+                    list(created.keys())[0] in ('gt', 'gte', 'lt', 'lte')
+                date = try_convert_to_int(list(created.values())[0])
+                assert type(date) is int and date > 1500000000
+            elif type(created) is str:
+                date = try_convert_to_int(created)
+                assert type(date) is int and date > 1500000000
+            elif created:
+                raise UserError(400, 'Bad request')
```
Comment (adrienverge) [inline localstripe/resources.py:491]: To be consistent with the rest of the code, I'd prefer
```python
assert type(created) in (dict, str)
```
on line 482.

Comment (H--o-l) [reply localstripe/resources.py:491]: Nice idea!
Here is the git diff I push to fix it:
```diff
--- a/localstripe/resources.py
+++ b/localstripe/resources.py
@@ -479,16 +479,14 @@ class Charge(StripeObject):
     def _api_list_all(cls, url, customer=None, created=None, limit=10):
         try:
             assert _type(customer) is str and customer.startswith('cus_')
+            assert type(created) in (dict, str)
             if type(created) is dict:
                 assert len(created.keys()) == 1 and \
                     list(created.keys())[0] in ('gt', 'gte', 'lt', 'lte')
                 date = try_convert_to_int(list(created.values())[0])
-                assert type(date) is int and date > 1500000000
             elif type(created) is str:
                 date = try_convert_to_int(created)
-                assert type(date) is int and date > 1500000000
-            elif created:
-                raise UserError(400, 'Bad request')
+            assert type(date) is int and date > 1500000000
         except AssertionError:
             raise UserError(400, 'Bad request')

**Comment by H--o-l** [comment]
Hey Adrien!

Thanks for the feedback.
I became aware this morning that I didn't implement the action of the `created : {'gt': timestamp}` parameter, thus I add three lines of code, and retested it:
```diff
@@ -500,6 +498,9 @@ class Charge(StripeObject):
 
         li = super(Charge, cls)._api_list_all(url, limit=limit)
         li._list = [c for c in li._list if c.customer == customer]
+        if created.get('gt'):
+            li._list = [c for c in li._list
+                        if c.created > try_convert_to_int(created['gt'])]
         return li
```
I also rebase the PR since you merge https://github.com/adrienverge/localstripe/pull/141 (thanks!)

**Comment by H--o-l** [comment]
Thanks!

**Pull request merged**